src/s1a_portapotty.tw2

```diff
@@ -151,25 +134,10 @@ At the bottom of the basin was a key.
 (click: ?tapTrigger)[(show: ?tapSection)]\
 (click: ?drainTrigger)[(show: ?drainSection)]
 
-::Jail Cell (Lit) - door <349,776>
-(if: $hasJailKey is 0)\
-[(if: $triedDoor is 0)[I grabbed the handle and [pulled]<trigger1|.](if: $triedDoor is 1)[The door to my cell was still [locked tight]<trigger1|.]]\
-(if: $hasJailKey is 1)[I slipped the key into the [keyhole]<trigger2|.]\
-\
-|lockedDoorSection)[
-
-(if: $triedDoor is 0)[(set: $triedDoor to 1)No good. The door wouldn't budge.](else:)[Still no good. I had to find a way to get it open.]
-
->> [[I glanced around the room again.|Jail Cell (Lit) - hub]] ]\
-\
-|unlockDoorSection)[The key was chipped and rusty. I wriggled it into the lock, forcing it in with my palm.
-
-After a moment, the key slid in.
+::Jail Cell (Lit) - basin - empty <70,885>
+The basin was empty. A reddish stain ran along the edge of the bowl.
 
-[[I turned it.|Jail Cell - exit]] ]\
-\
-(click: ?trigger1)[(show: ?lockedDoorSection)]\
-(click: ?trigger2)[(show: ?unlockDoorSection)]
+>> [[I took another look around.|Jail Cell (Lit) - hub]]
 
 ::Jail Cell - exit <345,1001>
 With a soft //thnck//, the lock turned.
```

Comment:
maybe add en or em dashes to either side of the onomatopoeia, calling it out even further? works either way tho!